acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -318,7 +318,7 @@ export default async function init(element) {
     })
   }
 
-  if (VERB === 'compress-pdf' || VERB === 'reorder-pages' || preRenderDropZone) {
+  if (window.browser?.isMobile && (VERB === 'fillsign' || VERB === 'sendforsignature' || VERB === 'add-comment' || VERB === 'crop-pages' || VERB === 'number-pages')) {
```

Comment:
need to leave the existing conditions as well
See comment on ticket, we may be able to extend this further

Reply:
@peller working on it as per the request in the ticket.

Reply:
Done